templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -24,7 +24,7 @@
  *}
 {block name='product_miniature_item'}
     <article class="product-miniature js-product-miniature mb-3" data-id-product="{$product.id_product}" data-id-product-attribute="{$product.id_product_attribute}">
-        <div class="card card-product">
+        <div class="card card-product thumbnail-container">
```

Comment:
could you add "card card-product /js thumbnail-container" plz ?

Reply:
just a mnemonic code (so we know that this css class has an interaction with js

Reply:
Ok, but `/js` is never used elsewhere. What I found to be used is `thumbnail-container js-thumbnail-container`, doubling the class, once with a `js-`-prefix